server/common/oursrc/scripts-proxy/statusz.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"errors"
+	"net"
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+)
+
+func nolvsPresent() bool {
+	if _, err := os.Stat("/etc/nolvs"); err == nil {
+		return true
+	}
+	return false
+}
+
+type HijackedServer struct {
```

Comment:
Why is this type called that?

Reply:
Because it's an http.Server that uses hijacked connections from the Proxy object to directly serve requests. ("Hijack" is a term established by net/http to mean "take over a Conn".) I added comments.

Reply:
The comment still doesn't make it clear what "hijack" means here.

Reply:
Reworded again.